eng/common/scripts/Package-Properties.ps1

```diff
@@ -10,28 +10,31 @@ class PackageProps
     [string]$ReadMePath
     [string]$ChangeLogPath
     [string]$Group
+    [string]$SdkType
```

Comment:
Might be a good idea to keep them in the same order. i.e. `SdkType` before `Group`